### PR TITLE
@dzucconi - mobile web tweaks

### PR DESCRIPTION
**Pull request by broskoski**
Re #435 

Covers everything with the exception of "leave ‘follow’ and ‘connect’ buttons visible on channel and profile blocks" which I can't figure out. Gonna try and take a pass at restructuring the block thumbnail (as its 3 different layers of absolutely positioned divs).

### CODE REVIEW

File: apps/block/stylesheets/index.styl [hunk 1]
```diff
@@ -202,26 +203,29 @@
...
 
   +below(950px)
     height 90vh
     width 10vw
     position absolute
     left 0
     top 0
+    display none
+    &:hover 
+      opacity 0
```
Comment (dzucconi) [inline apps/block/stylesheets/index.styl]: how does this have a hover state if it's already `display: none`?

File: apps/block/stylesheets/sidebar.styl [hunk 2]
```diff
@@ -1,4 +1,8 @@
+sb_font_size = 13px
+sb_line_height = 1.5
```
Comment (dzucconi) [inline apps/block/stylesheets/sidebar.styl]:
```styl
sidebar = {
  font_size: 13px,
  line_height: 1.5
}
```

File: apps/channel/client/channel_collaboration_view.coffee [hunk 3]
```diff
@@ -6,9 +6,7 @@ collaboratorsTemplate = -> require('../templates/collaborators/collaborators.jad
 
 module.exports = class ChannelCollaborationView extends Backbone.View
   initialize: ({ @isCollaboration, @isEditable, @channel })->
-    @collection.on "sync", @render, @
-    @collection.on "reset", @render, @
-    @collection.on "remove", @render, @
+    @collection.on "sync reset remove add update", @render, @
```
Comment (dzucconi) [inline apps/channel/client/channel_collaboration_view.coffee]:
```coffee
@listenTo @collection, 'sync reset remove add update', @render
```

Comment (dzucconi) [inline apps/channel/client/channel_collaboration_view.coffee]: (For easier teardown)

File: apps/channel/client/channel_edit_collaborators_view.coffee [hunk 4]
```diff
@@ -12,9 +12,7 @@ module.exports = class ChannelCollaborationView extends Backbone.View
     'click .collaborators__edit__result__remove' : 'removeCollaborator'
 
   initialize: ({ @isCollaboration, @isEditable, @channel })->
-    @collection.on "sync", @render, @
-    @collection.on "reset", @render, @
-    @collection.on "remove", @render, @
+    @collection.on "sync reset remove add", @render, @
```
Comment (dzucconi) [inline apps/channel/client/channel_edit_collaborators_view.coffee]:
```coffee
@listenTo @collection, 'sync reset remove add', @render
```

File: apps/channel/client/channel_new_collaborator_view.coffee [hunk 5]
```diff
@@ -22,7 +23,7 @@ module.exports = class newCollaboratorView extends Backbone.View
     @clear() unless $(e.target).closest('#metadata--collaborators').length
 
   addCollaborator: (e) ->
-    @collaborators._add $(e.target).data 'id'
+    @collaborators._add @results.get $(e.target).data 'id'
```
Comment (dzucconi) [inline apps/channel/client/channel_new_collaborator_view.coffee]: Why is this method underscored?

File: apps/tools/stylesheets/tools.styl [hunk 6]
```diff
@@ -1,39 +1,47 @@
...
+      margin 5px 0 10px 0
+
+    button
+      margin 20px 0 0 0
+
+    .small
+      font-size s_caption + 1
+      color textgray
+      margin 0 0 8px 0
+      line-height 14px
```
Comment (dzucconi) [inline apps/tools/stylesheets/tools.styl]: This is fine but in general it'd be nice if all these values weren't hard-coded and followed a particular scale for visual rhythm.

File: apps/user/templates/metadata_pockets/about.jade [hunk 7]
```diff
@@ -2,7 +2,7 @@ extends ../../../../components/meta_pocket/templates/meta_pocket
 
 block vars
   - title = "About"
-  - extraClasses = "is-readable"
+  - extraClasses = "is-readable show-on-mobile"
```
Comment (dzucconi) [inline apps/user/templates/metadata_pockets/about.jade]: What happened to BEM?

File: apps/block/stylesheets/sidebar.styl (hunk 2 above)
Comment (broskoski) [inline apps/block/stylesheets/sidebar.styl]: For some reason this breaks stylus...

File: apps/channel/client/channel_collaboration_view.coffee (hunk 3 above)
Comment (broskoski) [inline apps/channel/client/channel_collaboration_view.coffee]: 👍

File: apps/block/stylesheets/sidebar.styl (hunk 2 above)
Comment (dzucconi) [inline apps/block/stylesheets/sidebar.styl]: Sigh

File: apps/channel/client/channel_new_collaborator_view.coffee (hunk 5 above)
Comment (broskoski) [inline apps/channel/client/channel_new_collaborator_view.coffee]: The API for POSTing and DELETE-ing collaborators doesn't jive well with Backbone.

File: apps/tools/stylesheets/tools.styl (hunk 6 above)
Comment (broskoski) [inline apps/tools/stylesheets/tools.styl]: Agreed

File: apps/channel/client/channel_new_collaborator_view.coffee (hunk 5 above)
Comment (dzucconi) [inline apps/channel/client/channel_new_collaborator_view.coffee]: In which case it'd be preferable to overwrite the model `sync` method. But carry on for now.

File: apps/user/templates/metadata_pockets/about.jade (hunk 7 above)
Comment (broskoski) [inline apps/user/templates/metadata_pockets/about.jade]: I shall convert this to BEM

File: apps/block/stylesheets/sidebar.styl (hunk 2 above)
Comment (broskoski) [inline apps/block/stylesheets/sidebar.styl]: https://github.com/stylus/stylus/issues/1405

Comment (broskoski) [inline apps/block/stylesheets/sidebar.styl]: I've definitely seen you use this before..

**Comment by broskoski** [comment]
Why yarn always fail on travis?

**Comment by dzucconi** [comment]
I don't know. Feel free to delete the yarn lock and regenerate the NPM shrinkwrap for now though.

**Pull request merged**